### ISSUE TITLE
Log errors reporting TTS soundcard busy

### ISSUE THREAD

**Issue by riban-bw** [zynthian/zynthian-issue-tracking#1648]
# Describe the issue
There are errors in the log reporting that attempts to send a beep to the soundcard fail due to the card being busy.
# To Reproduce
## Steps to reproduce the behavior:
1. Do Sofware Update from admin maenu.
 ## Expected behaviour
Tone sounds periodically during update.
## Actual behaviour
No tone and errors logged.
# Additional context
Log:
For each expected beep:
```
ERROR:zynthian_tts._do_beep: TTS failed to send tone to soundcard - Device or resource busy [hw:Device]
```

I thought I had fixed this by putting the beep within the TTS thread task. Seems to need more attention.

# Configuration
## Hardware
```
Raspberry Pi 5 Model B Rev 1.0
Audio: Tascam US-16x08
Display: MIPI DSI 800x480 (inverted)
Wiring: TOUCH_ONLY
I2C: Not detected
Profile: v5
```
## System
```
Debian GNU/Linux 12 (bookworm)
Build Date: 2024-10-02
Memory: 14% (1158M/8063M)
Internal Storage: 30% (65G/234G ext4)
Temperature: 51.0ºC
Overclock: Maximum
```
## MIDI & UI
```
Tuning: 440 Hz
Master Channel: Off
Preload Presets: On
ZS3 (SubSnapShots): Off
Power Save: 10 minutes
Audio Levels on Snapshots: Off
```
## Software
```
zyncoder: vangelis (a4b603f) 
zynthian-ui: vangelis (7105ba5)
zynthian-sys: vangelis (3acc7da)
zynthian-data: vangelis (32d5cc5)
zynthian-webconf: vangelis (f55fba2)
```
